astropy/cosmology/io/table.py

```diff
@@ -11,20 +11,20 @@
 from .utils import convert_parameter_to_column
 
 
-def from_table(table, index=None, *, move_to_meta=False, cosmology=None):
+def from_table(table, index=None, *, move_to_meta=False, cosmology=None, rename=None):
     """Instantiate a `~astropy.cosmology.Cosmology` from a |QTable|.
 
     Parameters
     ----------
     table : `~astropy.table.Table`
         The object to parse into a |Cosmology|.
-    index : int, str, or None, optional
+    index : int or str or None, optional
         Needed to select the row in tables with multiple rows. ``index`` can be
         an integer for the row number or, if the table is indexed by a column,
         the value of that column. If the table is not indexed and ``index``
         is a string, the "name" column is used as the indexing column.
 
-    move_to_meta : bool (optional, keyword-only)
+    move_to_meta : bool, optional keyword-only
```

Comment:
Same with this: original just seems better

Reply:
This comes from https://numpydoc.readthedocs.io/en/latest/format.html#parameters, where the separation should be a comma.

Reply:
OK, that's fine. Here I mostly disliked "optional keyword-only" without comma. Do we need `keyword-only`? That duplicates the signature. Anyway, this one I mind much less.

Reply:
Hopefully codified in numpydoc eventually.
https://github.com/numpy/numpydoc/pull/367

Reply:
The lack of comma... yeah, same complaint as above... All this mixing of types and comments is harder to chunk than pre-chunked info!

```
move_to_meta : <type block>, <comment block>
```

vs

```
move_to_meta : mixed, type, and, comment, block
```

I'm not opposed to changing it, but I at least find it less useful than pre-chunking the info.

Reply:
I'll roll this back for now, and push a PR later that rectifies the docs.